### ISSUE TITLE
Использовать гендерно-нейтральный язык

### ISSUE THREAD

**Issue by Minoru** [NotDotTeam/trust#6]
В оригинальном тексте Ники избегает указаний на пол игрока — везде фигурируют нейтральные «other player» и тому подобное. Неплохо было бы это передать.

Но я понятия не имею, как это сделать ._.

**Comment by h8nor** [comment]
Хранилище ещё используется?!
В примечаниях опечатка "во предпоследнем раунде", и в новых лицах нечитаемый текст:
"привет я сначала сотрудничать. если ты тоже доверяться, я делать свой предыдущий ход, даже если это ошибка. если ты обманывать, я делать свой предыдущий ход наоборот, даже если это ошибка."
Сделать пул-реквест?

**Comment by toby3d** [comment]
@bopoh13 Репозиторий жив и иногда подтягивает изменения из родительского репозитория. По поводу опечаток - да, откройте PR, пожалуйста.

**Comment by h8nor** [comment]
По теме: "козлов" -> "сволочей".
Скорее, из-за особенностей русского языка, подобрать нейтральное слово будет невозможно. Например, если рассматривать субстантивацию в законотворчестве, то все слова будут иметь выраженный род (что далеко не всегда отражает гендерную принадлежность субъекта). Хм, интересно было бы узнать, как обстоят дела в немецком языке.

Задача игрока понимать и различать правила этой игры от прочих. Если игрок в силу своих физиологически особенностей не способен адекватно реагировать на установленные правила, - ему лучше заняться другой игрой (и не нарушать 1-й принцип честной игры: "Не мешай своими действиями другим").

**Comment by toby3d** [comment]
С моей точки зрения гендерной нейтральности крайне сложно добиться и проще взять мужской род слова "игрок" (который, кстати, женской формы не имеет или я долблюсь в глаза). Но я не истина в последней инстанции, потому оставляю этот issue открытым для дискуссий.

**Comment by bopoh13** [comment]
@toby3d, полностью солидарен.
Да, слово "игрок" в русском языке имеет мужской род, как "участник" или "человек".

**Comment by h8nor** [comment]
[Род](https://en.wikipedia.org/wiki/Grammatical_gender) субъекта в английском, японском и др. аналитических языках переходит в скрытую категорию (внутренняя флексия). В русском языке выделяют [грамматические категории](https://ru.wikipedia.org/wiki/%D0%93%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%B0%D1%8F_%D0%BA%D0%B0%D1%82%D0%B5%D0%B3%D0%BE%D1%80%D0%B8%D1%8F), но они не относятся ни к половым признаком, ни к _social construct of gender_. Задача не имеет решения.

**Comment by Minoru** [comment]
@bopoh13 Слова понимаю, а общий смысл уже нет ._. Попробую-ка привести ещё примеров, чтобы было понятно, зачем я открывал issue.

Например, на второй странице у нас написано: «Что должен сделать ты?» Такая формулировка подразумевает, что читатель — мужчина. Вопрос можно переформулировать и избежать любых предположений: «что следует сделать тебе?» или, пожалуй, более литературно — «как следует поступить тебе?».

На следующей странице написано «ты должен сжульничать», и это тоже легко переформулировать — «тебе следует сжульничать». Чуть дальше есть «забыл, кто есть кто?» — можно заменить на «не помнишь, кто есть кто?». «Ты всё ещё можешь быть скептичен» — «Возможно, ты всё ещё сомневаешься». «Поздравляю, ты поставил на правильную лошадку» — «Поздравляю, твоя ставка оказалась верна!». И так далее.

У меня нет лингвистического образования, так что не исключено, что мне просто повезло с примерами, а на самом деле не всякую фразу можно переписать на таком «гендерно-нейтральном» языке. Но попробовать сто́ит?

Сам я не готов этим сейчас заниматься, но, по крайней мере, хотел бы добиться удаления метки wontfix.

**Comment by toby3d** [comment]
Убедительно. Метку убрал.

**Comment by h8nor** [comment]
> «Что должен сделать ты?» Такая формулировка подразумевает, что читатель — мужчина. Вопрос можно переформулировать и избежать любых предположений: «что следует сделать тебе?»

Да, вы не поняли. С точки зрения русского языка у _таких_ одушевлённых имён существительных (в т.&nbsp;ч. личностных местоимений) [род](https://ru.wikipedia.org/wiki/%D0%A0%D0%BE%D0%B4_(%D0%BB%D0%B8%D0%BD%D0%B3%D0%B2%D0%B8%D1%81%D1%82%D0%B8%D0%BA%D0%B0)) общий, а не мужской. **Ты/тебе** это одно и то же слово в именительном и дательном падежах.

**Comment by Minoru** [comment]
К словам «ты/тебе» у меня никаких претензий нет. Претензии к словам «должен», «забыл», «скептичен», «поставил», имеющих мужской род.

**Comment by h8nor** [comment]
Я уже [объяснил](https://github.com/NotDotTeam/trust/issues/6#issuecomment-417009343) что проблема вами надумана.

**Comment by Minoru** [comment]
Что значит «надумана»? По-твоему, что я могу сказать девушке «ты должен написать реферат» или «ты пропустил нашу годовщину», и это будет совершенно корректно с точки зрения русского языка? Не верю.

**Comment by bopoh13** [comment]
Я не могу отвечать на много вопросов сразу. Давайте не будем пока уходить от первого примера.

В фразе «Что должен сделать ты?» слово "ты" является **одушевлённым именем существительного общего рода**. А вы утверждаете, что имя существительного "ты" имеет **мужской род**. Потому проблема и надумана.

**Comment by h8nor** [comment]
Я не могу отвечать на много вопросов сразу. Давайте не будем пока уходить от первого примера.

В фразе «Что должен сделать ты?» слово "ты" является **одушевлённым именем существительного общего рода**. А вы утверждаете, что имя существительного "ты" имеет **мужской род**. Глаголы относятся к именам существительных, - потому проблема и надумана.

**Comment by bopoh13** [comment]
> «Поздравляю, ты поставил на правильную лошадку»

Отчасти соглашусь. Вот здесь уже сложнее.

**Comment by h8nor** [comment]
> «Поздравляю, ты поставил на правильную лошадку»

Не пишите глаголы в прошедшем времени. Остальное можно перефразировать.

**Comment by Minoru** [comment]
А, я наконец понял, что тебя запутало, @bopoh13. [В первом своём комментарии](https://github.com/NotDotTeam/trust/issues/6#issuecomment-417885982) я одновременно менял глагол и имя существительное, не уточняя, в чём именно проблема. Позже я [написал](https://github.com/NotDotTeam/trust/issues/6#issuecomment-417912493), что дело в глаголах, но было уже поздно — ты решил, что я заменяю «ты» на «тебя», якобы считая, что у «ты» мужской род.

В общем, всё не так. Я борюсь только с глаголами, имеющими мужской род. Замены «ты» на «тебя» приходится делать просто для согласования.

**Comment by h8nor** [comment]
@Minoru, здо́рово, что мы пришли к взаимопониманию. Из текста было сложно понять суть проблемы.

**Comment by h8nor** [comment]
@bopoh13 бесцельно потраченое время на синтетическую модель игры :D
![9ee379_193df4ce6302457e97a87ac083068cdc](https://user-images.githubusercontent.com/6163937/210285955-f9356ee5-bd6d-4555-b71d-b5be40e3f2a4.gif)
